apps/configurator/lib/configurator/configuration/character.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Configurator.Configuration.Character do
+  @moduledoc """
+  Character schema
+  """
+  use Configurator.Schema
+  import Ecto.Changeset
+
+  schema "characters" do
+    field :active, :boolean, default: false
+    field :name, :string
+    field :base_speed, :decimal
+    field :base_size, :decimal
+    field :base_health, :integer
+    field :base_stamina, :integer
+
+    field :max_inventory_size, :integer
+    field :natural_healing_interval, :integer
+    field :natural_healing_damage_interval, :integer
+    field :stamina_interval, :integer
+
+    # TODO This should be removed once we have the skills relationship
```

Comment:
Let's link its issue here. If it doesn't have one, let's create it.

Reply:
Sure thing!

Reply:
Done